Rakefile

```diff
@@ -5,6 +5,14 @@ require "micro_migrations"
 
 ActiveRecord::Base.schema_format = :sql
 
+if ENV['LOGS_DATABASE'] == '1'
```

Comment:
Let's make these 2 separate rake tasks. One can be rake db:migrate the other can be rake db:migrate:logs. Then we can have 2 tests one for each rake task, and save on the env var. Also, I would add a third task that can be used rake db:migrate:all which calls both other rake tasks.